code/__DEFINES/magic.dm

```diff
@@ -0,0 +1,26 @@
+
+//schools of magic - unused for years and years on end, finally has a use with chaplains getting punished for using "evil" spells
```

Comment:
wait woah woah woah woah wooooooah

https://www.youtube.com/watch?v=zEtgRBjEZp0

Reply:
at least mention this in the PR body, dude

you should probably just atomize this into its own PR, really